docs/book/README.md

```diff
@@ -0,0 +1,23 @@
+# cachepot distributed compilation quickstart
```

Comment:
The title of this file should be about running a local instance of the book, not its content.

Reply:
title changed to `# How to run cachepot book?` which makes more sense.